docs/topics_5/web-create-project.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='add_project_{context}']
+= Creating a project
+
+You can create a project in the {WebName} with the *Create project* wizard.
+
+.Procedure
+
+. In the {WebName}, click *Projects*.
+. Click *Create project*.
+. Enter a unique name for the project, an optional description, and click *Next*.
++
+. To upload applications, click the *Upload* tab, click *Browse*, select the application files you want to upload, and click *Close*.
++
+Uploading applications stores them directly on the {ProductShortName} server.
+
+. To register a server path, click the *Server path* tab and enter the *Server-side path* of the application in the field.
++
+Registering the server path of an application ensures that {ProductShortName} always uses the latest version.
+. Click *Next*.
+. Click one or more transformation targets.
++
+.Transformation target screen
+image::web-configure-analysis_51.png[Transformation target screen]
+. Click *Next*.
+. Select packages and click '>' to include them in the analysis.
+. Click *Next*.
++
+. If you want to add a custom rule, click *Add rule*.
++
+See link:{ProductDocRulesGuideURL}[{RulesDevBookName}] for more information.
+
+. To upload a ruleset file, click the *Upload* tab, click *Browse*, select one or more files, and click *Close*.
++
+A ruleset file must have an `.mta.xml` or `.windup.xml` extension. The uploaded file is stored on the {ProductShortName} server.
++
+. To register the server path of a ruleset file, click the *Server path* tab, enter the *Rules* path, and click *Save*.
++
+Registering the server path ensures that the {ProductShortName} server always uses the latest version of the ruleset files.
++
+. Click *Next*.
++
+. If you want to add a custom label, click *Add label*.
+. To upload a labelset file, click the *Upload* tab, click *Browse*, select one or more files, and click *Close.*
```

Comment:
Is that the term used? Labelset? Never seen this before. WIsh we could provide more info